### PR TITLE
updating report api spec to account for source and error aggregations

### PR DESCRIPTION
**Pull request by jschatzman-lp**

### CODE REVIEW

File: pages/Documents/Reporting/OutboundReportingAPI/overview.md [hunk 1]
```diff
@@ -166,17 +167,26 @@ Click [**Account**](https://proactive-messaging.z1.fs.liveperson.com/api/api-doc
...
+ "handoffids": [
+   "H123456", "H987655"
+ ],
+  "source": [
+   "API", "UI", "LPWF"
+ ]
 }
 ```
 
+Each of the filter options shown above are optional. Regardless of what filter options are used, the results will always be grouped by channel, skill, and transactionday. When additional filters, such as source or messagstatus or handoffid are added, the results will be grouped by them as well and these will fields will be included in the response.
```
Comment (lp-gdoshi) [inline pages/Documents/Reporting/OutboundReportingAPI/overview.md:188]: @jschatzman-lp the second sentence is breaking and needs to be corrected.

File: pages/Documents/Reporting/OutboundReportingAPI/overview.md [hunk 2]
```diff
@@ -166,17 +167,26 @@ Click [**Account**](https://proactive-messaging.z1.fs.liveperson.com/api/api-doc
...
-     "billing"
-   ]
- }
+ "channels": [
+   "sms"
+ ],
+ "skills": [
+   "billing"
+ ],
+ "messagestatus": [
```
Comment (lp-gdoshi) [inline pages/Documents/Reporting/OutboundReportingAPI/overview.md:176]: @jschatzman-lp We dont have messagestatus in this account analytics api. Please can you recheck and update this

File: pages/Documents/Reporting/OutboundReportingAPI/overview.md [hunk 3]
```diff
@@ -383,14 +394,25 @@ Click [**Transaction**](https://proactive-messaging.z1.fs.liveperson.com/api/api
...
  "skills": [
-   "billing", "sales"
+   "billing"
+ ],
+ "messagestatus": [
+   "DELIVERED", "FAILED", "RESPONDED", "READ"
+ ],
+ "transactionids": [
+   "03cb1b10-2e46-4c10-bdc7-3d9c23654d51"
+ ],
```
Comment (lp-gdoshi) [inline pages/Documents/Reporting/OutboundReportingAPI/overview.md:407]: @jschatzman-lp Can we remove other things and just keep the changes related to the source field. The other changes for transaction API are already present in production.

**Review by lp-gdoshi — CHANGES_REQUESTED** [review]
There is some change requires in Analytics API as we dont support messageStatus filter.

File: pages/Documents/Reporting/OutboundReportingAPI/overview.md (hunk 1 above)
Comment (jschatzman-lp) [reply pages/Documents/Reporting/OutboundReportingAPI/overview.md]: @lp-gdoshi Fixed

File: pages/Documents/Reporting/OutboundReportingAPI/overview.md (hunk 3 above)
Comment (jschatzman-lp) [reply pages/Documents/Reporting/OutboundReportingAPI/overview.md:407]: @lp-gdoshi I'm not sure I understand why you want to remove this, I think the dev wiki should call out that we have an option to filter on transaction ID using transaction API

Comment (jschatzman-lp) [reply pages/Documents/Reporting/OutboundReportingAPI/overview.md:409]: @lp-gdoshi Nevermind, I see. We need to differentiate the request body between 1.0 and 2.0 for transactions api. I will fix this.

**Review by lp-gdoshi — APPROVED** [review]

**Comment by JSchatzman** [comment]
@itay1313 Can you please merge this?

**Pull request merged**